js/main.js

```diff
@@ -1049,70 +1048,64 @@ function StravaEnhancementSuite($, options) {
       });
     });
 
+    if (!Object.values(data).some(x => x.count > 1)) return; // No repeated segment effort, do not continue
```

Comment:
this is especially important. Previously, repeated segments table was added all the time, even when there were no repeated segments.